.vscode/settings.json

```diff
@@ -8,20 +8,9 @@
     "quickfix.biome": "explicit",
     "source.organizeImports.biome": "explicit"
   },
-  "[typescript]": {
-    "editor.defaultFormatter": "biomejs.biome"
-  },
-  "[json]": {
-    "editor.defaultFormatter": "biomejs.biome"
-  },
-  "[javascript]": {
-    "editor.defaultFormatter": "biomejs.biome"
-  },
-  "[jsonc]": {
-    "editor.defaultFormatter": "biomejs.biome"
-  },
-  "[typescriptreact]": {
+  "[typescript][json][javascript][jsonc][typescriptreact]": {
```

Comment:
This formatter settings are being overriden by L3.

Reply:
That's what I thought as well, but when I started using Biome, VS Code kept reverting to Prettier as the default formatter.

Reply:
> That's what I thought as well, but when I started using Biome, VS Code kept reverting to Prettier as the default formatter.

Seems I've found the issue here. The core problem stemmed from Biome not being installed globally on the system. This absence led to Biome's inoperability, causing VS Code to fall back to Prettier for all formatting operations. After installing Biome shows in this menu

![image](https://github.com/user-attachments/assets/da35be32-8a12-4104-bbcf-6b71fed9cc5c)

Here's a sample code snipppet to test Output - Prettier and Output - Biome respectively

```
function  testFunction ( a , b ) {
    if(a > b) {
return  a;
} else {
    return  b;
}
}
```

So, the problem was resolved by installing Biome globally. However, the "format on save" functionality is not working on my machine with this

I will play with this behavior a little bit and I'll be back with updates on this soon